apis/Google.LongRunning/Google.LongRunning.Snippets/OperationsClientSnippets.g.cs

```diff
@@ -53,9 +53,9 @@ public class GeneratedOperationsClientSnippets
             // Create client
             OperationsClient operationsClient = OperationsClient.Create();
             // Initialize request argument(s)
-            string formattedName = OperationsClient.FormatOperationPathName("[OPERATION_PATH]");
+            string name = "";
             // Make the request
-            Operation response = await operationsClient.GetOperationAsync(formattedName);
+            Operation response = await operationsClient.GetOperationAsync(name);
```

Comment:
I think it's here: https://github.com/googleapis/googleapis/blob/master/google/longrunning/longrunning_gapic.yaml#L17
Which specifies the BSD 3-clause license.

Pubsub is defined here: https://github.com/googleapis/googleapis/blob/master/google/pubsub/v1/pubsub_gapic.yaml#L21
Which specifies the Apache 2 license.